package.json

```diff
@@ -69,7 +69,7 @@
     ],
     "overrides": {
       "@types/node": "^18.16.16",
-      "chokidar": "3.5.2",
+      "chokidar": "^4.0.1",
```

Comment:
Can we move this to `catalog:`?

Reply:
this needs to be in `pnpm.overrides` because some dev dependencies are pulling on older versions, even though they work with 4.x

Reply:
Can we move the other two uses then?

Reply:
we unfortunately can't remove it from the nodes package, but I've move the version to catalog. 
but, for now at least, we need to keep the override as well.